tests/test_dsmc_handlers.py

```diff
@@ -450,7 +452,57 @@ def test_compress_archive_mini(self):
 
         self.assertTrue(os.path.exists(os.path.join(archive_path, "file.csv")))
         self.assertFalse(os.path.exists(os.path.join(archive_path, "file.bin")))
-        self.assertTrue(os.path.exists(os.path.join(archive_path, "directory2")))
+        self.assertFalse(os.path.exists(os.path.join(archive_path, "directory2")))
+        self.assertTrue(os.path.exists(os.path.join(archive_path, "directory3")))
         self.assertTrue(os.path.exists(os.path.join(archive_path, "testrunfolder_archive_tmp.tar.gz")))
 
         shutil.rmtree(archive_path)
+
+    def test_compress_archive_nothing_excluded(self):
+        """
+        Don't exclude anything
+        """
+        root = self.dummy_config["path_to_archive_root"]
+        archive_path = os.path.join(root, "testrunfolder_archive_tmp")
+        original = os.path.join(root, "testrunfolder_archive_input")
+
+        try:
+            shutil.rmtree(archive_path, ignore_errors=True)
+            shutil.copytree(original, archive_path)
+
+            # update the config to exclude nothing from archive
+            self.dummy_config = TestUtils.DUMMY_CONFIG
+            self.dummy_config["exclude_from_tarball"] = []
+
+            resp = self.fetch(
+                self.API_BASE + "/compress_archive/" + os.path.basename(archive_path),
+                method="POST",
+                allow_nonstandard_methods=True)
```

Comment:
I'm curious to know why this is necessary?

Reply:
not sure, I just built on the existing tests.. could it be that it allows for running tests is debugging mode without timing out? I think I got that problem when not using this argument

Reply:
I think maybe it could because it is a `POST` without a body, and I'm not sure that is "standard".

Reply:
yes, that's probably it